src/mjolnir/valhalla_add_predicted_traffic.cc

```diff
@@ -173,7 +173,7 @@ void update_tile(const std::string& tile_dir,
   size_t pred_count = 0;
   for (uint32_t j = 0; j < tile_builder.header()->directededgecount(); ++j) {
     auto found = speeds.find(j);
-    pred_count += found != speeds.cend() && found->second.coefficients.has_value();
+    pred_count += found != speeds.cend() && bool(found->second.coefficients);
```

Comment:
https://www.boost.org/doc/libs/1_68_0/libs/optional/doc/html/boost_optional/reference/header__boost_optional_optional_hpp_/detailed_semantics___optional_values.html

basically, `bool()` and `has_value` is the same thing

Reply:
yep i noticed this too. would you mind using `static_cast<bool>()` for these?

Reply:
yeah sure, but for me, `bool()` is a defined specialized  explicit operator from `boost::optional`(even in lower version) class... it's not a "just" casting :)